app/models/audit.rb

```diff
@@ -46,6 +46,16 @@ def self.since?(itemizable, *location_ids)
       .exists?
   end
 
+  def self.finalized_since?(itemizable, *location_ids)
+    item_ids = itemizable.line_items.pluck(:item_id)
+    where(status: "finalized")
+      .where(storage_location_id: location_ids)
+      .where(created_at: itemizable.created_at..)
+      .joins(:line_items)
+      .where(line_items: {item_id: item_ids})
+      .exists?
```

Comment:
Added a `finalized` claused based on @cielf feedback https://github.com/rubyforgood/human-essentials/pull/4086#discussion_r1481867310

Reply:
@cielf we should be able to just add `finalized` to the previous method, no? I don't see any reason to have both of them.

Reply:
I don't know of any reason we would need a since that wouldn't be finalized.   Are we using both?

Reply:
This adds a second version that only checks finalized audits - my assumption is that the first one was really incorrect and _also_ should only be checking finalized audits. So all we need to do is fix that one and don't add a new one.

Reply:
Yup.  That's my take.   An audit is really just a work in progress until it's finalized. IMO.

Reply:
Ok @dorner @cielf I modified the original only, but kept the `finalized_since` name as that is more explicit, and threw in an assertion for the change. Is it a safe change for the original use of `since?`? Should be a problem since we're only making things more safe/stricter right? And specs are green.